neanderthal/build.gradle

```diff
@@ -4,10 +4,11 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'com.getkeepsafe.dexcount'
+apply plugin: 'kotlin-kapt'
 
 buildscript {
     ext {
-        kotlinVersion = '1.0.4'
+        kotlinVersion = '1.3.41'
         supportLibraryVersion = '24.2.1'
```

Comment:
Better to update the support library version to be compatible with targetSdk : 28

Reply:
Done